ui/components/MesheryPatterns/lifecycle.js

```diff
@@ -0,0 +1,180 @@
+// import React, { useContext, useEffect, useState } from 'react';
+// import CheckIcon from '../../../assets/CheckIcon';
+// import DryRunIcon from '../../../assets/DryRunIcon';
+// import { DeploymentSelectorIcon } from '../../../assets/DeploymentSelectorIcon';
+//
+// import {
+//   ModalBody,
+//   ModalFooter,
+//   useStepper,
+//   CustomizedStepper,
+//   ModalButtonPrimary,
+//   ModalButtonSecondary,
+//   Box,
+//   Stack,
+//   Checkbox,
+//   Typography,
+// } from '@layer5/sistent';
+//
+// export const ValidateContent = {
+//   btnText: 'Next',
+//   cancel: true,
+// };
+//
+// const ValidateDesignStep = ({ handleClose }) => {
+//   return <ValidateDesign handleClose={handleClose} />;
+// };
+//
+// const SelectTargetStep = ({ setSharedData, sharedData, ...props }) => {
+//   const SelectDepylomentTarget = injectedReactComponents.get(
+//     GLOBAL_COMPONENTS.SelectDeploymentTarget,
+//   );
+//
+//   return (
+//     <Box>
+//       <SelectDepylomentTarget />
+//     </Box>
+//   );
+// };
+//
+// const DryRunStep = ({ handleClose, bypassDryRun, setBypassDryRun, setDryRunErrors }) => {
+//   console.log('Rendering DryRunStep', bypassDryRun, setBypassDryRun, setDryRunErrors);
+//
+//   useEffect(() => {
+//     console.log('Mounting DryRunStep');
+//   }, []);
+//   const handleErrors = (errors) => {
+//     setDryRunErrors(errors);
+//   };
+//   const bypassValidation = bypassDryRun || false;
+//   const toggleBypassValidation = () => setBypassDryRun(!bypassDryRun);
+//
+//   return (
+//     <Box>
+//       <DryRunDesign handleClose={handleClose} handleErrors={handleErrors} />
+//       <Stack spacing={2} direction="row" alignItems="center">
+//         <Checkbox value={bypassValidation} onChange={toggleBypassValidation} />
+//         <Typography variant="body2">Bypass errors and initiate deployment</Typography>
```

Comment:
you can pass the sistent typography variant

Reply:
makes sense :+1: